notebooks/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>Waiting to have our own datasets :)</p><p>(cause was thinking about "how can we take this code to a dataset_utils file, and then just call one function", and then understood that it is redundant.)</p><p>However we need to add some credit for uci in notebook, no?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>